### PR TITLE
Prevent forwarding of content-disposition to couch : ref #238

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: server/lib/binary.coffee [hunk 1]
```diff
@@ -29,6 +29,8 @@ module.exports.addBinary = (doc, attachData, readStream, callback) ->
                 db.merge doc._id, binary: binList, (err) ->
                     log.error err if err?
                     cb()
+
+        delete readStream.headers['content-disposition']
```
Comment (nono) [inline server/lib/binary.coffee]: @aenario it looks weird. Are you sure of your change?

I don't think we have headers on a stream, and even if it was the case, I don't see how they can be transfered to Couchdb (`pipe` doesn't do that). Maybe I'm missing something.

Comment (aenario) [inline server/lib/binary.coffee]: So, 

- this stream comes from multiparty, which does in fact copy the part headers to the stream.headers : https://github.com/andrewrk/node-multiparty/blob/1104e3b8334d5b74cb4bf6a3b6331d0ef86afcf6/index.js#L505

- And is piped to request, which does copy the header from its source field on('pipe')
https://github.com/request/request/blob/1b19c88134b54fec75a270aeb8847d465a4ba8b1/request.js#L519-L525

Comment (aenario) [inline server/lib/binary.coffee]: It does however breaks when the stream is not from multiparty as in thumbs management (see travis). 
image

Comment (nono) [inline server/lib/binary.coffee]: OK, thanks for the explanation!

**Comment by clochix** [comment]
Tested by Pierre, seems to fix #238

**Pull request merged**